### PR TITLE
Include B. Dixon direct comment

### PR DESCRIPTION
**Pull request by bam241**
This aims to fix the comments of B Dixon on the report.

I'll try to catch the "missing words, singular and plural mismatches, etc."

BaM

### CODE REVIEW

File: EG23.tex [hunk 1]
```diff
@@ -265,16 +268,26 @@ \subsection{Cooling/Storage}
     \label{tab:cooling_1}
 \end{table}
 
+The specified cooling time is supposed to be 60 month and a year each for
+reprocessing and fabrication. 84 month of cooling time have been implemented
```
Comment (gonuke) [inline EG23.tex]: Don't start sentence with a number.  Either rearrange sentence or write out as "Eighty-four"

File: EG23.tex [hunk 2]
```diff
@@ -265,16 +268,26 @@ \subsection{Cooling/Storage}
...
+because both separation and fabrication requires any buffer time. However, a
```
Comment (gonuke) [inline EG23.tex]: "because neither separation nor fabrication model any processing time."

File: EG23.tex [hunk 3]
```diff
@@ -265,16 +268,26 @@ \subsection{Cooling/Storage}
...
+time step is required to send the flows from the cooling to the separation and
+from the separation to the fabrication and then from the fabrication to the
+reactor. In those Cyclus simulations, the  out of reactor minimum time is 87 months.
```
Comment (gonuke) [inline EG23.tex]: "minimum time out of reactor is 87 months."

Comment (gonuke) [inline EG23.tex]: I guess we could redo the simulation with 81 month storage??

File: EG23.tex [hunk 4]
```diff
@@ -265,16 +268,26 @@ \subsection{Cooling/Storage}
...
 deployed in 2030 and one in 2040. In Case 1, the SFR separation facilities have
 an effectively infinite separation capacity, but the separation rate has been
-throttled indirectly using the gradual fuel fabrication capacity growth. 
-As the separation output buffers are not limited, those separation facilities will
-reprocess all the used available fuel with a constant reprocessing rate.
+throttled indirectly using the gradual fuel fabrication capacity growth.  As the
+separation output buffers are not limited, those separation facilities will
+reprocess all the used available fuel with a constant reprocessing rate. For the
+FBR separation the constant reprocessing capacity is so high ($5000\times12
+tHM/year$) that it will not limits the reprocessing. The used FBR-MOX fuel will
```
Comment (gonuke) [inline EG23.tex]: "limits" => "limit"

File: EG23.tex (hunk 3 above)
Comment (bam241) [inline EG23.tex]: Yes, should I run it ?

**Comment by bam241** [comment]
@gonuke should I merge this one too ?

**Comment by gonuke** [comment]
Yes - it's good to go.

**Pull request merged**